@xen-orchestra/backups/formatVmBackups.js

```diff
@@ -0,0 +1,8 @@
+const mapValues = require('lodash/mapValues')
+
+const { formatVmBackup } = require('./_formatVmBackup')
```

Comment:
IMHO, you can inline it for now :slightly_smiling_face:

Reply:
Where?

Reply:
In this file, no need to keep it separate, it's used nowhere else.